### ISSUE TITLE
Expo-notifications, App closed notifications are not showing.

### ISSUE THREAD

**Issue by jpenjgusic** [expo/expo#9163]
### Summary of Issue 
On android standalone build, when app is closed push notifications are not showing up.
I'm using expo-notifications ~0.3.3 in managed workflow.

### Environment - output of `expo diagnostics` & the platform(s) you're targeting
Expo CLI 3.21.13 environment info:
    System:
      OS: Windows 10 10.0.18362
    Binaries:
      Node: 10.16.0 - C:\Program Files\nodejs\node.EXE
      Yarn: 1.22.4 - C:\Users\juric\AppData\Roaming\npm\yarn.CMD
      npm: 6.9.0 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: Version  3.4.0.0 AI-183.6156.11.34.5522156
    npmPackages:
      expo: ~38.0.8 => 38.0.8
      react: ~16.11.0 => 16.11.0
      react-dom: ~16.11.0 => 16.11.0
      react-native: https://github.com/expo/react-native/archive/sdk-38.0.2.tar.gz => 0.62.2
      react-native-web: ~0.11.7 => 0.11.7

### Reproducible Demo
```json
{
  "expo": {
    "version": "1.0.8",
    "name": "Demo",
    "scheme": "demo.com",
    "orientation": "portrait",
    "slug": "demo-slug",
    "icon": "./assets/images/logo.png",
    "privacy": "unlisted",
    "splash": {
      "image": "./assets/images/splash.png",
      "resizeMode": "cover",
      "backgroundColor": "#ffffff"
    },
    "updates": {
      "fallbackToCacheTimeout": 0
    },
    "assetBundlePatterns": [
      "**/*"
    ],
    "ios": {
      "supportsTablet": true
    },
    "web": {
      "favicon": "./assets/favicon.png"
    },
    "notification": {
      "icon": "./assets/images/notification-logo.png",
      "color": "#3FC1C9"
    },
    "android": {
      "versionCode": 8,
      "package": "com.demo.package",
      "googleServicesFile": "./google-services.json",
      "useNextNotificationsApi": true,
      "config": {
        "googleMaps": {
          "apiKey": "xxxxxx"
        }
      }
    }
  }
}
```
 
### Expected Behavior vs Actual Behavior
When app is foregrounded notification handler successfully handles notification.
When app is background notifications are shown in notification tray but notification icon is missing.
When app is closed notification are not shown in notification tray.
If I remove "useNextNotificationsApi" in app.json everything is working correctly.
Apk was tested on Xiaomi Redmi note 8

**Comment by lukmccall** [comment]
Hi @jpenjgusic, the bug with the notification icon should be fixed by https://github.com/expo/expo/pull/9116. 

Unfortunately, I wasn't able to reproduce the bug with the notification tray.  Could you provide a simple demo app?

**Comment by jpenjgusic** [comment]
Can't reproduce this.
So, closing issue.

**Comment by amber-pli** [comment]
I thinks #9037 is related to our issue and fixed in #9287, which will be available in SDK 39
